pkg/api/types.go

```diff
@@ -378,7 +380,7 @@ func (p *PullPolicy) String() string {
 
 // Type implements the Type() function of pflags.Value interface
 func (p *PullPolicy) Type() string {
-	return "string"
+	return "PullPolicy"
```

Comment:
This is used to tell [pflag](https://godoc.org/github.com/spf13/pflag) the name of the type of a custom flag.
See the example in https://godoc.org/github.com/spf13/pflag#example-package.

Calling it "string" in all three cases seems a bit misleading, but it does change how the default values are printed in the command-line help. Perhaps it was intentional to leverage the treatment pflag gives to strings...
[spf13/pflag/blob/master/flag.go#L502-L511](https://github.com/spf13/pflag/blob/cb88ea77998c3f024757528e3305022ab50b43be/flag.go#L502-L511)

I got here originally via `goconst` suggesting that `"string"` should be a constant because it appears 3 times...

I will revert the changes to the output of the `Type` methods, no changes to CLI help messages.
@php-coder thanks for asking!